scripts/classes/class.Extrausers.php

```diff
@@ -53,12 +53,21 @@ private static function _generateFile($file, $query, &$cronlog)
 			switch ($type) {
 				case 'passwd':
 					$line = $u['username'] . ':' . $u['password'] . ':' . $u['uid'] . ':' . $u['gid'] . ':' . $u['comment'] . ':' . $u['homedir'] . ':' . $u['shell'] . PHP_EOL;
+					$permissions = 0644;
+					$user = 'root';
```

Comment:
Doesn't hurt, either. Since you're not unlinking it if it exists but only touching it if it does not, it's possible that it pre-exists belonging to something != root:root...